src/pint/scripts/tcb2tdb.py

```diff
@@ -0,0 +1,55 @@
+"""PINT-based tool for converting TCB par files to TDB."""
+
+import argparse
+
+from loguru import logger as log
+
+import pint.logging
+from pint.models import get_model
+from pint.models.tcb_conversion import convert_tcb_to_tdb
+
+pint.logging.setup(level="INFO")
+
+__all__ = ["main"]
+
+
+def main(argv=None):
+    parser = argparse.ArgumentParser(
+        description="PINT tool for converting TCB par files to TBD.",
+        formatter_class=argparse.ArgumentDefaultsHelpFormatter,
+    )
+    parser.add_argument("input_par", help="Input par file name (TCB)")
+    parser.add_argument("output_par", help="Output par file name (TDB)")
+
+    args = parser.parse_args(argv)
+
+    welcome_message = """This script converts TCB par files to TDB.
+    Please note that this conversion is not exact and the timing model 
+    should be re-fit to the TOAs. 
+
+    The following parameters are converted to TDB:
+        1. Spin frequency, its derivatives and spin epoch
+        2. Sky coordinates, proper motion and the position epoch
+        3. Keplerian binary parameters
+    
+    The following parameters are NOT converted although they are 
+    in fact affected by the TCB to TDB conversion:
+        1. Parallax
+        2. TZRMJD and TZRFRQ
+        3. DM, DM derivatives, DM epoch, DMX parameters
+        4. Solar wind parameters
+        5. Binary post-Keplerian parameters including Shapiro delay 
+           parameters
+        6. Jumps and DM Jumps
+        7. FD parameters
+        8. EQUADs
+        9. Red noise parameters including FITWAVES, powerlaw red noise and 
+           powerlaw DM noise parameters
+    """
+    log.info(welcome_message)
+
+    model = get_model(args.input_par, allow_tcb=True)
+    convert_tcb_to_tdb(model)
+    model.write_parfile(args.output_par)
+
+    log.info(f"Output written to {args.output_par}.")
```

Comment:
I suggest changing this to `log.warning` and including words about how the output is only approximately correct. I know that is in the help, but I think a user should see that every time it runs.

Reply:
There is a "welcome_message" that is shown every time the script runs, and has this information. I have changed that to a warning.